python/dolfin/mesh.py

```diff
@@ -43,6 +43,15 @@ def __new__(cls, value_type, mesh, dim=None):
             return mvc(mesh)
 
 
+class Mesh(cpp.mesh.Mesh):
+    def __init__(self, mpi_comm, cell_type, points, connectivity,
+                 global_cell_indices, ghost_mode, num_ghost_cells=0,
+                 custom_permutation=[]):
```

Comment:
Can this be made `custom_permutation=[]`?

Reply:
What do you mean? isn't that what it says?

Reply:
Sorry, I meant to write `custom_permutation=None`.

Reply:
No, it can't causes type-error in the wrapper.